R/net.mod.infection.R

```diff
@@ -92,8 +92,12 @@ infection.net <- function(dat, at) {
 
       # Set new infections vector
       idsNewInf <- unique(del$sus)
-      dat$attr$status[idsNewInf] <- "i"
-      dat$attr$infTime[idsNewInf] <- at
+      status.temp <- get_attr(dat, "status")
+      status.temp[idsNewInf] <- "i"
```

Comment:
This seems unnecessary. status and infTime can be pulled at the top of the function, and then set at the bottom of the function. Each module should follow this convention:

1. Pull from dat (param, attr, etc., needed in processes)
2. Work on vectors directly
3. Put back on dat

Reply:
Overall workflow of get/set functions updated throughout modules.